src/conn/conn_compact.c

```diff
@@ -8,8 +8,9 @@
 
 #include "wt_internal.h"
 
+#define WT_BACKGROUND_COMPACT_SLEEP_TIME 60
```

Comment:
I think this is actually used as an upper bound for how long compact on a file can be skipped. Would:

`WT_BACKGROUND_COMPACT_MAX_SKIP_TIME`

Seem like a better name?

Reply:
Yeah that sounds good.

Reply:
Changed here https://github.com/wiredtiger/wiredtiger/pull/9541/commits/112ccd9c97af7114df00e6b89c772d59b5d29444.